tools/generator/src/Generator/SetTargetConfigurations.swift

```diff
@@ -424,7 +424,7 @@ $(CONFIGURATION_BUILD_DIR)
                     ])
                 }
 
-                if !target.isSwift, target.inputs.containsSourceFiles {
+                if target.inputs.containsSourceFiles {
```

Comment:
Should this continue to check for isSwift?

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/2106